framework/auth/views.py

```diff
@@ -48,10 +49,10 @@ def reset_password(auth, **kwargs):
         }
         raise HTTPError(400, data=error_data)
 
-    if request.method == 'POST' and form.validate():
+    if request.method == 'POST':  # TODO - do we need more than KO password validation?
```

Comment:
Though if it's possible to circumvent the front-end and `POST` directly to this endpoint, then yes.